catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -160,7 +175,13 @@ private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable>
                 operationPolicy.putAll(policy);
             }
         }
-        return new PolicyResponseImpl(operationPolicy, operationPolicy);
+        if (securityAttributes.isEmpty()) {
+            return new PolicyResponseImpl(operationPolicy, operationPolicy);
+        } else {
+            Map<String, Set<String>> itemPolicy = new HashMap<>();
+            itemPolicy.putAll(securityAttributes);
```

Comment:
It looks like we create the securityAttributes map for every request anyways. Is there a reason why we dump it into a new map here? Couldn't we just pass securityAttributes as the item policy?